### ISSUE TITLE
The link to the pretrained model seems to be broken

### ISSUE THREAD

**Issue by Rso0902** [JasonQSY/Articulation3D#1]
Thank you for sharing this remarkable work. However, the link to the pretrained model seems to be broken. Could you please let me know how I can access it?

**Comment by JasonQSY** [comment]
Yes it's gone since I'm no longer at University of Michigan. Here is a temporary link to download: https://www.dropbox.com/scl/fi/9kau684y2woz3f3xqon63/model_final.pth?rlkey=tqgedkhj2jm76bbpkct1kvxv5&st=ugeuextu&dl=0

Let me know if it works for you.
